app/src/components/ItemsNav.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import { ItemsHeader, BoldText } from '@/styles/StyleHomePage';
+
+export default function ItemsNav() {
+    const [value, setValue] = React.useState<string | undefined>();
```

Comment:
Just wondering, does it need an initial value? If not, it's fine as-is :)

Reply:
I don't think it needs an initial value, the link to the Home Page is on the logo 'Cl311', and all the items on 'ItemsNar' (Dashboard, Add Report, Report List, etc.) lead to different webpages